src/lib/components/buttons/CopyButton.svelte

```diff
@@ -22,7 +20,7 @@
 <button
   type="button"
   data-testid="copy-button"
-  class={clazz}
+  class="btn variant-ringed-primary {$$props.class || ''}"
```

Comment:
Lets allow full customization of the css

```suggestion
  class={$$props.class ?? 'btn variant-ringed-primary hover:variant-ghost-primary'}
```